src/main/java/frc/robot/Autonomous.java

```diff
@@ -41,42 +36,54 @@
 
     private final PIDController positionPIDController;
 
+    private final CANSparkMax leftSide;
+    private final CANSparkMax rightSide;
+
     public Autonomous(Waiter waiter, Drive drivetrain, BallPath ballPath){
         this.waiter = waiter;
         this.drivetrain = drivetrain;
+        this.leftSide = drivetrain.getLeftSide();
+        this.rightSide = drivetrain.getRightSide();
         this.ballPath = ballPath;
         this.positionPIDController = new PIDController(kP, kI, kD);
         this.positionPIDController.setTolerance(DRIVE_DIST_TOLERANCE);
     }
 
-    double calcTicks(double encoderTicks){
-        // Calculates ticks per revolution(shaft rotation)
-        double gearRatio = 8;
-
-        double ticks = encoderTicks / gearRatio * wheelCircumference; // ticksPer cancels so it's just encoderTicks / gearRatio
+    public void turn(AHRS gyro, double degree) {
+        gyro.reset();
+        double error = degree - gyro.getAngle();
+        double kP = 0.005;
```

Comment:
Why not use an actual `PIDController` instance here, instead of inlining a `P-Controller` implementation within this method?

Reply:
The math seemed simple enough to implement myself, but it can be changed if need be. Would rather not change it as turning seems to be working fine currently.